src/Catalyst.Core.Lib/DAO/DaoBase.cs

```diff
@@ -28,24 +28,10 @@
 
 namespace Catalyst.Core.Lib.DAO
 {
-    public abstract class DaoBase<TProto, TDao> : IMapperInitializer,
-        IValueConverter<TProto, TDao>
+    public abstract class DaoBase
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Convert this 'abstract' class to a concrete class with a protected constructor.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4404007)